src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return (days * 40 - 50);
+  }
+
+  if (days >= 3) {
+    return (days * 40 - 20);
+  } else {
+    return (days * 40);
```

Comment:
create a variable with this value, because you use it several times

Reply:
Done.